ui/user_settings.ts

```diff
@@ -24,56 +27,56 @@ const ExchangeBadge = ({ name, css_class }) => `
 </div>
 `;
 
-function show_loading(selector) {
+function show_loading(selector?: any) {
     $('html').addClass('wait');
     if (selector) {
         $(selector).attr('disabled', 'disabled');
     }
 }
 
-function stop_show_loading(selector) {
+function stop_show_loading(selector?: any) {
     $('html').removeClass('wait');
     if (selector) {
         $(selector).removeAttr('disabled');
     }
 }
 
-function disable_api_entry(selector_text, value) {
+function disable_api_entry(selector_text: any, value: any) {
```

Comment:
isn't a jquery selector a `string` here and also `value` a string as it's just html text to fill in?